packages/datatrak-web/src/layout/UserMenu/MenuList.tsx

```diff
@@ -180,6 +201,15 @@ export const MenuList = ({
             </MenuListItem>
           ))}
       </Menu>
+      <CancelConfirmModal
+        headingText="Unsynced data"
+        bodyText="You are about to log out with unsynced data! Go back to your home page and sync using the top right sync button and sync before logging out"
```

Comment:
Ignore if this is designed copy, I think this message could be clearer:

```suggestion
        bodyText={<>You have unsynced changes, which will be lost if you log out now. To sync now, go to DataTrak Home and tap <SyncIcon />.</>}
```